design/FY2025/NFP-new-air-water-heat-pump.md

```diff
@@ -0,0 +1,1468 @@
+An Air-to-Water Heat Pump Model
+================
+
+**Yujie Xu, Tianzhen Hong**
+
+**Lawrence Berkeley National Laboratory***
+
+ - Original Date: February, 2025
+ - Final Date: March, 2025
+
+## Justification for a feature update ##
+
+Heat pumps (HP) are an essential technology for decarbonizing buildings.
+EnergyPlus is capable of modeling several types of heat pumps, e.g., for space
+cooling or heating: air to air, air to water, water to water; and heat pump
+water heaters. Although there is an existing feature to model air-to-water heat
+pumps in EnergyPlus, its capability is limited and its implementation is not
+straightforward compared with other types of HP. Some of the limitations
+include: Cannot specify parameters like single, multi-speed, or variable speed
+for capacity control (for example, the Daikin model EWYT-CZ series is inverter
+controlled, and the EWYQ series is a fixed-speed two-stage model.) Missing
+details of crankcase heater control and more flexible defrost control The naming
+of the object is not intuitive - it does not refer to water to water or air to
+water, making searching for the object difficult Missing the input of an
+availability schedule (which disables the HP for seasonal consideration)
+Designed as separate heating and cooling objects while in reality, it should be
+one piece of equipment
+
+This feature will create a new air-to-water heat pump (AWHP) model that includes
+heating and cooling components in one single object, capable of multi-speed
+controls, and includes more equipment details such as fan characteristics, PLR
+ranges, availability schedules, etc.
+
+## Overview ##
+
+### Introduction ###
+
+Air-to-water heat pumps are becoming increasingly popular in the U.S., with an
+expected annual growth of 15.5% in market share from 2023 to 2032 [1]. The
+following are some commercial AWHP products from one manufacturer. The AWHP can
+provide heating, cooling, and hot water at up to 60C (140F). The unit has
+variable speed control (inverter).
+
+<img src="awhp-figure-1.png" alt="awhp-figure-1" width="400px">
+</img>
+
+Figure 1. Schematic diagram of an AWHP system
+
+<img src="awhp-table-1-1.png" alt="awhp-table-1-1" width="400px">
+</img>
+<img src="awhp-table-1-2.png" alt="awhp-table-1-2" width="400px">
+</img>
+
+Table 1. A sample air-to-water heat pump product from Daikin (EWYT-CZ series) [2].
+
+### Existing approaches ###
+
+Currently, EnergyPlus uses a pair of objects, HeatPump:PlantLoop:EIR:Heating and
+HeatPump:PlantLoop:EIR:Cooling, with "Condenser Type" set to "AirSource", to
+model air-to-water heat pump as part of a central plant cooling and/or heating
+loop, which traditionally includes chillers to produce chilled water and a
+boiler to produce hot water. The following is an example of the current object.
+The object specifies capacity and EIR as functions of temperature and part load
+ratio, with three performance curves.
+
```

Comment:
Is this new object intending to replace the use of any existing objects and deprecate those? If not, adding a new object just seems to add some additional confusion on which one should be used. Perhaps you should include in the NFP some user oriented guidance on when each object that is mentioned in the NFP should be used including the proposed new one.

Reply:
The new object can model multi-speed/variable-speed/modular products while the existing one can't. So for these cases users would want to use the new object. I will add some documentation on how to choose between them.

Reply:
I added a section with notes on which model to use to model an air-to-water heat pump.